packages/next/pages/_document.tsx

```diff
@@ -566,6 +594,22 @@ export class NextScript extends Component<OriginProps> {
     })
   }
 
+  getPreNextScripts() {
+    const { scriptLoader } = this.context
+
+    return (scriptLoader.eager || []).map((file: string) => {
```

Comment:
I think we need to give this a scarier name than `eager` if it really means "`beforeHydration`".

Reply:
WE've gone back and forth on a few names before we settled on `eager`
Do you have any recommendation?